lib/tapioca/generators/todo.rb

```diff
@@ -0,0 +1,74 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Tapioca
+  module Generators
+    class Todo < Base
+      sig { override.void }
+      def generate
+        build_todos
+      end
+
+      sig { override.params(error: Error).void }
+      def handle_error(error)
+      end
+
+      sig { void }
+      def build_todos
+        todos_path = config.todos_path
+        compiler = Compilers::TodosCompiler.new
+        name = set_color(todos_path, :yellow, :bold)
+        say("Compiling #{name}, this may take a few seconds... ")
+
+        # Clean all existing unresolved constants before regenerating the list
+        # so Sorbet won't grab them as already resolved.
+        File.delete(todos_path) if File.exist?(todos_path)
+
+        rbi_string = compiler.compile
+        if rbi_string.empty?
+          say("Nothing to do", :green)
+          return
+        end
+
+        content = String.new
+        content << rbi_header(
+          "#{Config::DEFAULT_COMMAND} todo",
+          reason: "unresolved constants",
+          strictness: "false"
+        )
+        content << rbi_string
+        content << "\n"
+
+        outdir = File.dirname(todos_path)
+        FileUtils.mkdir_p(outdir)
+        File.write(todos_path, content)
+
+        say("Done", :green)
+
+        say("All unresolved constants have been written to #{name}.", [:green, :bold])
+        say("Please review changes and commit them.", [:green, :bold])
+      end
+
+      sig { params(command: String, reason: T.nilable(String), strictness: T.nilable(String)).returns(String) }
+      def rbi_header(command, reason: nil, strictness: nil)
```

Comment:
It's used in multiple places, should we move it in the base class?

Reply:
I'm going to handle that in the next smaller PR where I can extract some shared logic to `Base` where it makes sense, as well as more appropriate classes.

As you mentioned in https://github.com/Shopify/tapioca/pull/496#discussion_r707401996, this would probably be better suited in `Rbi::File` as an extension or something to that effect.

But I wanna keep the scope of this PR focused and not start making too many out of scope changes.